src/kvstore/NebulaStore.cpp

```diff
@@ -1061,9 +1042,8 @@ bool NebulaStore::checkLeader(std::shared_ptr<Part> part, bool canReadFromFollow
 void NebulaStore::cleanWAL() {
     folly::RWSpinLock::ReadHolder rh(&lock_);
     SCOPE_EXIT {
-        cleanWalWorker_->addDelayTask(FLAGS_clean_wal_interval_secs * 1000,
-                                      &NebulaStore::cleanWAL,
-                                      this);
+        storeWorker_->addDelayTask(
```

Comment:
`cleanWalWorker_` is looks more clear?

Reply:
Previously `cleanWalWorker_` will be good, it will add an extra work to refresh available space of each data path, so rename it.